### ISSUE TITLE
Hero: Pagination to load more requests "Click for next 5 requests"

### ISSUE THREAD

**Issue by gakshay** [gakshay/hookhook#31]
![screen shot 2015-05-04 at 12 17 36 pm](https://cloud.githubusercontent.com/assets/116975/7449789/a923b7ac-f25a-11e4-8460-01e994d86a6e.png)

**Comment by sanjusoftware** [comment]
@gakshay should it rather not be an infinite scroll.. lkke how FB offers.. the "click link" thing is too old now a days

**Comment by gakshay** [comment]
Yeah upon us on how to implement it, it can be just a scroll to load more requests...

**Comment by guptapiyush** [comment]
The thought behind is that if I receive many requests regularly and if I can browse them all, I might scroll through and respond to only few and leave others hanging. Following challenges arise for us
1. How to get users to take some action on each request? 
   Even if it means that they want to reject a request or do not want to respond. This is completely fine. This will help us acknowledge back to the requestor that we will not be able to facilitate this request. Some acknowledgment is better than none.
2. How to avoid request overload for high-in-demand users?
   If it is an infinite scroll, I'm pretty sure that the attention span that I will have for each request will be very small.

So how can we solve these challenges?
1. Limiting the scope to few requests helps in bringing focus to few requests at a time. This also reduces their cognitive load because of tons of requests.
2. Everyone gets curious about new requests and we all want to discover who, why, and what kind of requests that we receive. If we do not allow the users to see more requests until they have taken some action on previous 5 requests, they will build the habit of responding to each request. Of course we need to be cognizant of their challenges that they may face because of this constraint and fine-tune the process while listening to them.

Let's experiment with this and see how it gets perceived.
